### ISSUE TITLE
[BUG] Ensure file permissions are RW for needed files

### ISSUE THREAD

**Issue by RickJWagner** [code-ready/crc#639]
### General information

  * OS: macOS 
  * Did you run `crc setup` before starting it (Yes/No)? Yes

This ticket is the result of a user discussion, the usual requested artifacts are no longer available.  Sorry for that.

Problem:  CRC would not start because permissions were 'R', not 'RW' for directories holding:
/etc/resolver/testing
/etc/hosts

When the user made the directories 'RW', CRC started.

**Comment by RickJWagner** [comment]
SA Bill Novak has more information if needed.

**Comment by cfergeau** [comment]
This is already fixed in git
Duplicate of #618
